internal/model/app.go

```diff
@@ -10,7 +13,13 @@ type AppConfig struct {
 
 // GroupConfig is the group configuration.
 type GroupConfig struct {
-	Priority *int
+	Priority   *int
```

Comment:
Why an int pointer?

Reply:
To be able to see when the value has been set or not (`nil` vs !`nil`), because we can't rely on `0` being a non-set value, the user could want to set `0` as priority (more knowing that the defualt priority is `1000`).